cmd/config/web-burner-cluster-density/node_density_pod_served.yml

```diff
@@ -0,0 +1,49 @@
+---
```

Comment:
Can this filename go with cluster_density* prefix to keep the naming consistent across the directory.